README.md

```diff
@@ -94,7 +94,7 @@ Configure MLSpace using Option 2 if:
 - you wish to have your configuration changes in a file that's committed to git
 - will have to resolve conflicts when upgrading MLSpace
 
-If you are pre-creating roles you will need to ensure that the required role arns (`APP_ROLE_ARN`, `NOTEBOOK_ROLE_ARN`, `SYSTSTEM_ROLE_ARN`, `ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN`, `JOB_INSTANCE_CONSTRAINT_POLICY_ARN`), role names (`KEY_MANAGER_ROLE_NAME` if `EXISTING_KMS_MASTER_KEY_ARN` is not set), and `AWS_ACCOUNT` (used to ensure unique S3 bucket names) have been properly set in `lib/constants.ts`.
+If you are pre-creating roles you will need to ensure that the required role ARNs (`APP_ROLE_ARN`, `NOTEBOOK_ROLE_ARN`, and `SYSTSTEM_ROLE_ARN`), policy ARNs ( `ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN`, `JOB_INSTANCE_CONSTRAINT_POLICY_ARN`, and `KMS_INSTANCE_CONDITIONS_POLICY_ARN`), role names (`KEY_MANAGER_ROLE_NAME` if `EXISTING_KMS_MASTER_KEY_ARN` is not set), and `AWS_ACCOUNT` (used to ensure unique S3 bucket names) have been properly set in `lib/constants.ts`.
```

Comment:
> have been properly set in `lib/constants.ts`.

Could be good to clarify >have been properly set using the config wizard or in `lib/constants.ts`

Reply:
Yeah that wording is a little vague. Perhaps just a slight change like below is more clear? Or did you have something more substantial in mind?

> If you are pre-creating roles, you will need to ensure that the required role ARNs (`APP_ROLE_ARN`, `NOTEBOOK_ROLE_ARN`, and `SYSTEM_ROLE_ARN`), policy ARNs (`ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN`, `JOB_INSTANCE_CONSTRAINT_POLICY_ARN`, and `KMS_INSTANCE_CONDITIONS_POLICY_ARN`), role names (such as `KEY_MANAGER_ROLE_NAME` if `EXISTING_KMS_MASTER_KEY_ARN` is not set), and `AWS_ACCOUNT` (used to ensure unique S3 bucket names) have values defined for them in `lib/constants.ts`.

Reply:
I just specifically wanted to call out that these things can be set in the constants.ts file or by using the wizard, they don't *have* to put these in constants.ts (or at least not all of them).